technical_overview/technical_details/Message_flow.md

```diff
@@ -0,0 +1,60 @@
+# Message flow through EMAP pipeline
+As detailed in the [technical overview](../Technical_overview_of_EMAP.md), the purpose of the Experimental Medicine
+Application Platform (EMAP) is to offer real-time clinical data suitable to analysis, such as the prediction of ICU 
+bed occupancy based on current workloads and planned surgeries. This requires the pipeline to consolidate historical
+data with live information available through message streams. 
+
+Messages on these message streams hold varying kinds of data, including patient details such as name and medication 
+given, but also doctors treating the patient and which ward and bed this patient has been in. 
+It should be noted here, that the HL7 Reader in its current version supports different versions of the HL7 messaging 
+standard through HAPI. 
+
+The complexity of consolidating data originating from historic database and live streams is amplified by the fact that 
+the format of data differs depending on where it originates from. To overcome this limitation, an EMAP interchange data
+format has been defined to which all the data is transferred to in order to compare and align it. For further 
+information in the exchange format, please take a look [here](Interchange_format.md). 
+
+As not everyone might be familiar with the nature of the data in a hospital setting, additional information on the 
+nature of the available data in a hospital can be found [here](../Background_information.md)
+
+
+## Data from different sources 
+
+To illustrate the process of how data is merged as part of the EMAP pipeline, we have split this section into two parts,
+corresponding to historic data and data from messages streams as both are handled through different parts of the 
+EMAP pipeline components before being merged. Once all the data is accumulated in 
+[RabbitMQ](../technologies_used/RabbitMQ.md), all the data is handled in the same way, independent of the source it 
+originally came from.
+
+### From historic database to RabbitMQ
+
+The Hoover (cf. Figure 2 in [Technical Overview](../Technical_overview_of_EMAP.md)) runs detailed queries to receive data 
+held in historic database of the hospital wrt. the information about a patient that is relevant, e.g. historic encouters of 
+patients with medical staff. This information is then parsed into the interchange data format and written to the
+[RabbitMQ](../technologies_used/RabbitMQ.md) handling queue.
```

Comment:
```suggestion
The Hoover (cf. Figure 2 in [Technical Overview](../Technical_overview_of_EMAP.md)) runs  queries to extract data from hospital databases.
This can be historic data, or non-live data that is updated in a daily dump. 
This information is parsed into the interchange data format and published to the 
[RabbitMQ](../technologies_used/RabbitMQ.md) databaseExtracts queue.
```